coverage/src/BamView-WebComponent.js

```diff
@@ -134,6 +151,25 @@ class BamViewChart extends HTMLElement {
         this._meanCoverage = null;
     }
 
+    get options() {
+        const raw = this.getAttribute('options');
```

Comment:
I probably wouldn't bother supporting attributes for something this complicated (a property object should be fine), but I don't think we need to remove it since you already did the extra work to implement it.